front/src/pages/Plans.jsx

```diff
@@ -1,13 +1,35 @@
-import React, { useState, useEffect } from 'react';
-
+import React, {useState, useEffect} from 'react';
 
 export const Plans = () => {
 
+    function sayHello() {
+        console.log("Hello");
+    }
+
     return (
         <div className="container">
-            <div className="container__content">
-                <h1>Plans</h1>
-            </div>
+            <h1>Plans</h1>
+
+            <br/><br/>
+
+            <button className="btn btn--primary btn--sm" onClick={sayHello}>Default</button>
+            <button className="btn btn--primary btn--sm" onClick={sayHello}>Pressed</button>
+            <button className="btn btn--primary btn--sm" onClick={sayHello} disabled>Disabled</button>
+
+            <br/><br/>
+
+            <button className="btn btn--primary btn--sm btn--icon" onClick={sayHello}>Default</button>
+            <button className="btn btn--primary btn--sm btn--icon" onClick={sayHello}>Pressed</button>
```

Comment:
there className are to complicated - icon class should be adapt to separate component